types/koa-session/koa-session-tests.ts

```diff
@@ -0,0 +1,8 @@
+import * as Koa from 'koa';
+import * as session from 'koa-session';

```

Comment:
Please read https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this 

The correct syntax for this import would be:
```ts
import session = require('koa-session');
```

since ```koa-session``` is not a true es6 module. Your syntax happens to work here because of the extra ```declare module session``` in the declaration file, but this would be unnecessary as long as the correct import syntax is used.